src/components/Home.jsx

```diff
@@ -26,24 +30,27 @@ function Home() {
 
   return (
     <>
-      <h2>{currentPageLabel}</h2>
+      <h2 className="current-page-label">{currentPageLabel}</h2>
       <div className="all-the-articles">
         {articles.map((article) => {
           return (
             <>
               <div
                 data-articlename={article.title}
+                data-articleid={article.article_id}
                 className="individual-article"
                 onClick={handleClick}
               >
                 <img
                   className="individual-article-image"
                   src={article.article_img_url}
                 ></img>
-                <h3>{article.title}</h3>
-                <p class="article-text">Author: {article.author}</p>
-                <p class="article-text">Created: {article.created_at}</p>
-                <p class="article-text">Topic: {article.topic}</p>
+                <div className="article-info">
+                  <h3 class="article-title">{article.title}</h3>
+                  <p class="article-text">Author: {article.author}</p>
+                  <p class="article-text">Created: {article.created_at}</p>
```

Comment:
It would be better to have the timestamp formatted in a bit more of a human-readable way, than the default. Have a think about how you could manipulate the data yourself to do this or look into some packages that can do it for you 💡